src/java.base/share/classes/module-info.java

```diff
@@ -374,6 +374,7 @@
         java.prefs;
     exports sun.util.resources to
         jdk.localedata;
+    exports jdk.internal.constant;
```

Comment:
This is probably unintentional, if you do export we probably want to only export to select modules.

Reply:
Yes, seems IDEA took some liberties on a move refactor. Removing.